Makefile

```diff
@@ -17,6 +17,7 @@ format: FORCE
 
 test: lint FORCE
 	pytest -v -n auto test/
+	FUNSOR_NORMALIZE=1 pytest -v test/test_cnf.py
```

Comment:
let's remove the `NORMALIZE` option to reduce our feature flag set. i.e. normalize by default.